README.md

```diff
@@ -362,6 +362,33 @@ See more on [this blog post](https://medium.com/@tomchentw/imagemin-lint-staged-
 
 ## Frequently Asked Questions
 
+### Can I use `lint-staged` via node?
+
+Yes!
+
+```js
+const lintStaged = require('lint-staged')
+
+try {
+  const success = await lintStaged()
```

Comment:
I wonder if it would better still if failed tasks also rejected. `lintStaged` could even emit different errors based on the reason of rejection. Still, this is fine by me!

Reply:
I'd like that approach, but would want to move all the console logging into the executable so that the library can have finer control of the output. Differentiating between tasks failing and errors with configuration does feel important though.